packages/grafbase-sdk/src/define.ts

```diff
@@ -0,0 +1,141 @@
+import { Enum, EnumShape } from './enum'
+import { Input, InputFields } from './input_type'
+import { Interface, InterfaceFields } from './interface'
+import { Query, QueryInput } from './query'
+import { Type, TypeFields } from './type'
+import { EnumDefinition } from './typedefs/enum'
+import { InputDefinition } from './typedefs/input'
+import { ReferenceDefinition } from './typedefs/reference'
+import { Union } from './union'
+
+export default {
+  /**
+   * Creates a new composite type.
```

Comment:
"Object type" would be closer to the standard imo, if I'm understanding the purpose right. But just an opinion, no need to change.

Reply:
Agree on this.  The spec does use "composite type" in a few places, but its [definition includes input objects](http://spec.graphql.org/October2021/#sec-The-__Type-Type.Input-Object) rather than just objects

Reply:
I used the existing comment here but don't mind changing it

Reply:
Done